common/src/db/multi_model.rs

```diff
@@ -56,6 +62,37 @@ impl<E: EntityTrait> SelectIntoMultiModel for Select<E> {
         self.try_columns_prefixed(&prefix, O::Column::iter())
     }
 
+    fn try_model_columns_from_alias<O: EntityTrait>(
+        mut self,
+        _entity: O,
+        table_alias: &str,
+    ) -> Result<Self, DbErr> {
+        let prefix = format!("{table_alias}$");
+        for simple_col in O::Column::iter() {
```

Comment:
just making observation that this is the kind of code one sees buried deep in an ORM ... I understand sea_orm is minimal but I wonder if there are any existing tools there - a cursory scan there I could not find anything.

Reply:
Yah, this is leveraging SeaORM, but allowing us to take a wide join query and split into N different models, where out-of-the-box, it takes one row, and makes one model.